scene/animate-images-with-image-overlay/src/main/resources/animate_images_with_image_overlay.fxml

```diff
@@ -25,34 +25,17 @@
 <?import javafx.scene.paint.Color?>
 <?import javafx.scene.control.Label?>
 <StackPane fx:controller="com.esri.samples.animate_images_with_image_overlay.AnimateImagesWithImageOverlayController"
-           xmlns:fx="http://javafx.com/fxml" stylesheets="/style.css">
+           xmlns:fx="http://javafx.com/fxml" stylesheets="/style.css" styleClass="stack-pane">
     <SceneView fx:id="sceneView"/>
-    <VBox StackPane.alignment="BOTTOM_CENTER" maxWidth="-Infinity" maxHeight="50" styleClass="panel-region" alignment="TOP_CENTER">
+    <VBox StackPane.alignment="BOTTOM_CENTER" styleClass="panel-region" alignment="TOP_CENTER">
     <StackPane.margin>
         <Insets bottom="20"/>
     </StackPane.margin>
-    <padding>
-        <Insets topRightBottomLeft="20"/>
-    </padding>
-        <background>
-            <Background>
-                <fills>
-                    <BackgroundFill>
-                        <fill>
-                            <Color opacity="0.3"/>
-                        </fill>
-                    </BackgroundFill>
-                </fills>
-            </Background>
-        </background>
         <GridPane hgap="30" vgap="10" maxWidth="500">
```

Comment:
Just found a couple of resource for CSS in fxml:
https://docs.oracle.com/javafx/2/layout/style_css.htm
https://docs.oracle.com/javafx/2/api/javafx/scene/doc-files/cssref.html#gridpane

You could get rid of the rules here by adding to the css:
```
Grid Pane {
	-fx-max-width: 500;
    -fx-hgap: 30;
    -fx-vgap: 10;
}
```

It looks like we could get away with removing the `styleClass="stack-pane"` and `styleClass="panel-region"` (there's no other stack-pane that we need to differentiate that one from in the fxml file). Might be worth trying to remove them and see how the CSS reacts.

I think moving the 'min/maxWidth' parameters to the CSS is definitely ok, it's more cosmetic rather than any direct relevance to the code. I think it'd be good to highlight the `fx:id` parts and the method calls (`onAction="#handleframesComboBoxInteraction"`), so let's bring those to the front of the node declarations.

Looks like you can also define the alignment within the StackPane in the CSS file:
https://docs.oracle.com/javafx/2/api/javafx/scene/doc-files/cssref.html#stackpane

You could even declare most of the params for the slider:
https://docs.oracle.com/javafx/2/api/javafx/scene/doc-files/cssref.html#slider

only thing I can't see in there is whether we can control the element's position inside the grid pane (i.e. replace `GridPane.rowIndex="0"` etc.)

I'll let you decide when too much is too much 😄

Reply:
ok, I've done what I can with the css! thanks for the helpful links.